### PR TITLE
Fix api url to version 2 json array

### PR DESCRIPTION
**Pull request by sergynyo**

### CODE REVIEW

File: Hyber/MessageSender.php [hunk 1]
```diff
@@ -9,7 +9,7 @@
 class MessageSender
 {
     const API_HOST = "https://api.hyber.im";
-    const MESSAGE_CREATE_PATH = "/%s/messages/create";
+    const MESSAGE_CREATE_PATH = "/%s/messages/v2/create";
```
Comment (arodiss) [inline Hyber/MessageSender.php]: @sergynyo 
should be "v2/%s"

Comment (sergynyo) [inline Hyber/MessageSender.php]: to replace the creation?

Comment (arodiss) [inline Hyber/MessageSender.php]: @sergynyo 
currently on prod messages are created at `api.hyber.im/1111`
in new version will be `api.hyber.im/v2/1111`
Your URI is wrong

**Comment by sergynyo** [comment]
@arodiss 
done

**Pull request merged**